test/e2e-go/features/transactions/lease_test.go

```diff
@@ -196,7 +196,7 @@ func TestOverlappingLeases(t *testing.T) {
 
 	// construct transactions for sending money to account1 and account2
 	// from same sender with identical lease, but different, overlapping ranges
```

Comment:
Overlapping ranges yield several different scenarios. However, this test is not testing all of these generated scenarios. It is testing one, and when sporadically another one of the scenarios plays out, the test is failing. 

Changing the values in this commit, merely favors the tested scenario, but keeps the other scenarios less likely to play out, and more importantly, untested. 

To make sure the overlapping ranges is fully tested, we must enumerate all the possible scenarios (in the comments),  and show which test item is covering each of these possible scenarios.

Reply:
This is true, and we can add more tests, but the purpose of this PR is just to reduce the flakiness of the test we have now to stop breaking builds. We can add more tests in a separate PR